hi-ml-azure/testazure/testazure/test_azure_util.py

```diff
@@ -1399,9 +1398,21 @@ def test_get_run_source(dummy_recovery_id: str,
             assert isinstance(script_config.run, str)
 
 
+def delete_existing_blobs(datastore: AzureBlobDatastore, prefix: str) -> None:
+    """Deletes all existing files in blob storage at the location that the test uses.
+
+    param datastore: The datastore from which the files should be deleted.
+    param prefix: The prefix string for the files that should be deleted.
+    """
+    container = datastore.container_name
+    existing_blobs = list(datastore.blob_service.list_blobs(prefix=prefix,
+                                                            container_name=container))
+    for existing_blob in existing_blobs:
+        datastore.blob_service.delete_blob(container_name=container, blob_name=existing_blob.name)
```

Comment:
Should we add a check here to make sure that deletion was successful?

Reply:
Wouldn't an unsuccessful deletion simply raise an error? Not sure it's worth adding extra complexity here. Could be that the deletion is not immediately visible, for example.